docs/source/markdown/podman-run.1.md

```diff
@@ -837,6 +838,8 @@ Security Options
 - **seccomp**=_profile.json_:  Allowed syscall list seccomp JSON file to be used as a seccomp filter
 - **proc-opts**=_OPTIONS_ : Comma separated list of options to use for the /proc mount.  More details
   for the possible mount options are specified at **proc(5)** man page.
+- **unmask**=_ALL_ or _/path/1:/path/2_: Paths to unmask separated by a colon. If set to **ALL**, it will
+  unmask all the paths that are masked by default.
```

Comment:
Can we mention which paths are marked by default? May be worth adding a comment in the code as well to keep the man page and the `[]string` in sync.

Reply:
Can you also explain what a masked path is? Users may not be familiar?

Reply:
Fixed